conference.js

```diff
@@ -1967,7 +1967,7 @@ export default {
             (authEnabled, authLogin) =>
                 APP.store.dispatch(authStatusChanged(authEnabled, authLogin)));
 
-        room.on(JitsiConferenceEvents.PARTCIPANT_FEATURES_CHANGED,
+        room.on(JitsiConferenceEvents.PARTICIPANT_FEATURES_CHANGED,
```

Comment:
This change will not work. As the typo is in lib-jitsi-meet ... needs to be fixed there first ...

Reply:
Let me remove this fix here, since I think this kind of coupling error